src/route.js

```diff
@@ -27,11 +29,24 @@ var Route = function (routes, uri) {
 Route.prototype = {
 
   /**
+  Attempt to match the uri, or call a notFound handler if nothing matches.
+
+  @method matchOrNotFound
+  @param {Object} routeLevel
+  **/
+  matchOrNotFound: function (routeLevel) {
+    this.match(routeLevel);
+
+    if (!this.fullMatch && this.notFound) {
```

Comment:
Alternatively, we could build into Aviator a special redirect key.  Maybe that's too prescriptive though.